notebooks/02_model/mmlspark_lightgbm_criteo.ipynb

```diff
@@ -0,0 +1,449 @@
+{
```

Comment:
<p>We don't need the parentheses around ComputeModelStatistics().</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/680/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
oh, i add them when I want to chain operations in python, to me it's prettier than appending a slash to every line